cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/DiffController.java

```diff
@@ -68,4 +72,45 @@ public Integer call() throws Exception {
             return 0;
         }
     }
+
+    @Command(name = "space-developers", description = "Print the differences between " +
+            "the space developers given in the yaml file and the space developers of your cf instance.")
+    static class DiffSpaceDevelopersCommand implements Callable<Integer> {
+
+        @Mixin
+        LoginCommandOptions loginOptions;
+
+        @Mixin
+        YamlCommandOptions yamlCommandOptions;
+
+        @Override
+        public Integer call() throws Exception {
+            Log.info("Diffing space-developer(s)...");
+            DefaultCloudFoundryOperations cfOperations = CfOperationsCreator.createCfOperations(loginOptions);
```

Comment:
Should we move most of the Code to the DiffLogic class?
--> For example:
diffLogic = new DiffLogic();
String output = diffLogic.diffSpaceDevs(loginOptions, yamlCommandOptions);

If we would refactor the code like this, it could be tested way easier with JUnit.

Reply:
We might even want to pass the command options as parameters to the DiffLogic constructor, as they will be needed for various diff operations